### DISCUSSION TITLE
Fluoride, flouride or fluride?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#398] [Galaxy Trivia] [answered]
**Hint:** 🦷🪥

**Comment by peterbaccaro** [comment]
**Fluoride** is the correct spelling for the naturally occurring mineral used in many dental products to strengthen tooth enamel and prevent cavities.

**Comment by Roberto-Larivera** [comment]
Fluoride è il termine corretto

**Comment by thuongtruong1009** [comment, marked as answer]
The correct spelling is "fluoride".
